### PR TITLE
feat/COM-132 rewrite storing source code

### PR DESCRIPTION
**Pull request by SireeKoolenWijkstra**
Rewrote storing source code to allow for new format. Also took into account Benny van de Hoogen his suggestion for improving the standard.

### CODE REVIEW

File: docs/general/storing-source-code.md [hunk 1]
```diff
@@ -1,44 +1,43 @@
...
-- It makes collaboration easy.
-- It provides version control, allowing us to track all changes to the repository. 
+## When and for whom is this standard?
+This standard applies to front-end and back-end developers.<br/>
+This standard must be applied to all new repositories of the city of Amsterdam (new since May 2024).
 
-## Where to store your code
-The following guidelines can be used to determine where and how to store your code:
+## Public or private {#status}
+Infra-as-code logic must always be stored in a private repository. This improves transparancy and reusability, but protects us from exposing security implications that could benefit potential bad actors.
```
Comment (bennyvdhoogen) [inline docs/general/storing-source-code.md:15]: Typo, should be `transparency`, but my bad I guess!

**Review by bennyvdhoogen — APPROVED** [review]

File: docs/general/storing-source-code.md [hunk 2]
```diff
@@ -1,44 +1,43 @@
...
-### Github vs Azure Repos
-1. **Infra-as-code** that is used to deploy infrastructure and/or applications to our Azure Public Cloud is always stored in Azure Repos, close to the pipelines that deploy the infra and/or applications.
-    > The Cloudplatform (the team responsible for the Azure Cloud) requires infrastructure code to be private and stored on Azure Repos, allowing them to enforce authorization rules. 
+## Recommendations
+- Send an e-mail to the Datapunt team to get access to the Amsterdam organisation in GitHub. Your e-mail must include the following:
+  - your GitHub username
+  - your first and last name
+  - the team responsible for the project
+  - and your Product Owner needs to be included in het CC. 
+- Don't store data in your repository unless it's testing data.
```
Comment (thomasm0) [inline docs/general/storing-source-code.md:23]: Can we add `Data used with tests should always be anonymized`.

**Review by thomasm0 — APPROVED** [review]

File: docs/general/storing-source-code.md [hunk 3]
```diff
@@ -1,44 +1,43 @@
...
+- Require a pull request before merging
+  - Require approvals
+    - The required number of approvals before merging is at least 1
 
-Using Git has multiple benefits:
-- It allows us to be transparent when storing code publicly. 
-- It makes collaboration easy.
-- It provides version control, allowing us to track all changes to the repository. 
+## When and for whom is this standard?
+This standard applies to front-end and back-end developers.<br/>
```
[the remark this answers is not in the document]
Comment (SireeKoolenWijkstra) [reply docs/general/storing-source-code.md:11]: fixed it.

File: docs/general/storing-source-code.md (hunk 2 above)
Comment (SireeKoolenWijkstra) [reply docs/general/storing-source-code.md:23]: Added it. Also made it a little stronger by saying it must be anonymized

**Pull request merged**